### DISCUSSION TITLE
Qbittorrent, Radarr, Sonarr LXC Helper Script Mount Points

### DISCUSSION THREAD

**Discussion by hauserpro** [community-scripts/ProxmoxVE#532] [Q&A] [answered]
I can mount an SMB share to an LXC container no problem. Done it for some other containers. However while trying to install the **ARR LXC containers using the Helper Scripts I noticed there is no reference to what directory should I mount the SMB too so it shows up in the container for use?
Specifically after the mount points information for Qbittorrent, Radarr and Sonarr LXC containers? 

Example:
pct set 150 -mp0 /nas_mounted_folders/Media,mp=????

Thanks in advance

**Comment by Aloe-recite** [comment, marked as answer]
If I understand your question correctly, it's up to you choosing the mount point and configuring it in each app. Personal choice. Personally, I have /downloads and /movies, /video, /tv-shows, /documentaries, etc., of course radarr can only access the first two, sonarr all but "movies", etc.

**Reply by hauserpro** [reply]
Thank you. I think I confused myself by thinking the LXC Containers has a docker style folder structure where you have the right side of the colon (:) which you not supposed to change and the left which you can change. In LXC containers this principle does not apply.

**Reply by Aloe-recite** [reply]
you're welcome, been there and done the same not long ago! Glad you're up and running!

**Comment by knario** [comment]
Hi, I'll tell you how I have it in case it helps.
In the host (Proxmox) I have:
/mnt/downloads/ with different subfolders for each container, Qbittorrent, Amule, alldebrid, Telethon downloader, etc...
  And then /media with subfolders for each category, Books, series, Software, etc....
In each container I mount the folder I'm interested in and I do it in the same path of the host, so I don't go crazy to know in which folder is such and such a thing. 

I'll give you an example in case you need it. 😃

mp0: /mnt/download,mp=/mnt/download
mp1: /media/courses,mp=/media/courses
mp10: /media/animation,mp=/media/animation
mp11: /personal,mp=/personal
mp12: /media/music,mp=/media/music

**Reply by hauserpro** [reply]
> Hi, I'll tell you how I have it in case it helps. In the host (Proxmox) I have: /mnt/downloads/ with different subfolders for each container, Qbittorrent, Amule, alldebrid, Telethon downloader, etc... And then /media with subfolders for each category, Books, series, Software, etc.... In each container I mount the folder I'm interested in and I do it in the same path of the host, so I don't go crazy to know in which folder is such and such a thing.
> 
> I'll give you an example in case you need it. 😃
> 
> mp0: /mnt/download,mp=/mnt/download mp1: /media/courses,mp=/media/courses mp10: /media/animation,mp=/media/animation mp11: /personal,mp=/personal mp12: /media/music,mp=/media/music

thank you!!
